### PR TITLE
tests: fix the issue where all the tests were executed on secboot system

### PR DESCRIPTION
**Pull request by sergiocazzolato**
The idea is to run in ubuntu with secure-boot enabled the smoke tests and the tests just tagged to be executed on a secure-boot

### CODE REVIEW

**Comment by sergiocazzolato** [comment]
@zyga I see these errrors in the logs: 
2020-05-20 12:58:18 Cannot allocate google:ubuntu-20.04-64: google: error getting credentials using well-known file (/home/ubuntu/.config/gcloud/application_default_credentials.json): open /home/ubuntu/.config/gcloud/application_default_credentials.json: permission denied

File: tests/regression/lp-1813365/task.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 summary: Regression check for https://bugs.launchpad.net/snapd/+bug/1813365
-systems: [ubuntu-*, ubuntu-core-*, debian-*]
+systems: [ubuntu-1*, ubuntu-2*, ubuntu-core-*, debian-*]
```
Comment (zyga) [inline tests/regression/lp-1813365/task.yaml:2]: This looks valid but is it related?

File: tests/main/uc20-create-partitions-encrypt/task.yaml [hunk 2]
```diff
@@ -82,7 +82,7 @@ execute: |
         ./gadget-dir "$LOOP"
 
     echo "Check that the key file was created"
-    test "$(stat --printf=%s ./keyfile)" = 1095
+    test "$(stat --printf=%s ./keyfile)" -ge 1000
```
Comment (zyga) [inline tests/main/uc20-create-partitions-encrypt/task.yaml:85]: What's the idea here?

File: tests/main/uc20-create-partitions-encrypt/task.yaml [hunk 3]
```diff
@@ -72,8 +72,8 @@ execute: |
     LOOP="$(cat loop.txt)"
 
     echo "Run the snap-bootstrap tool"
-    go get ../../../lib/create-partitions
-    create-partitions \
+    go get ../../lib/uc20-create-partitions
+    uc20-create-partitions \
```
Comment (zyga) [inline tests/main/uc20-create-partitions-encrypt/task.yaml:76]: Was this broken before?

File: tests/regression/lp-1813365/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/regression/lp-1813365/task.yaml:2]: If the system is ubuntu-* then ubuntu-secureboot-20.04-64 also matches and we don't want that, other tests are filtering like this (ubuntu-1*, ubuntu-2*) already and I copied from them.

File: tests/main/uc20-create-partitions-encrypt/task.yaml (hunk 2 above)
Comment (cmatsuoka) [reply tests/main/uc20-create-partitions-encrypt/task.yaml:85]: The size of the sealed key is still unstable and I'm testing against a reasonable minimum size.

File: tests/main/uc20-create-partitions-encrypt/task.yaml (hunk 3 above)
Comment (cmatsuoka) [reply tests/main/uc20-create-partitions-encrypt/task.yaml:76]: `create-partitions` was renamed to `uc20-create-partitions` in 0dc0dd6 but the test was not updated accordingly, causing a stale package error.

File: tests/regression/lp-1813365/task.yaml (hunk 1 above)
Comment (zyga) [reply tests/regression/lp-1813365/task.yaml:2]: Ah, I msised that the original didn't have a digit prefix.

**Pull request merged**